.github/workflows/ci_pipeline.yml

```diff
@@ -173,7 +173,7 @@ jobs:
   user_firedrake_tests:
     runs-on: ubuntu-latest
     container:
-      image: firedrakeproject/firedrake-vanilla:2025-01
+      image: firedrakeproject/firedrake-vanilla:latest
```

Comment:
That is dangerous. You don't know what the next stage will bring.. can you pin the version to something?

Reply:
un-pinning the version is precisely the point of this PR. The Gusto pipeline runs on the latest Firedrake, so if we pin Firedrake but not Gusto, we might get into trouble. If we pin both, we make sure that the current state runs now, but people may not be able to install the current Firedrake outside of containers in the future.
I suggest we test on the latest Firedrake and revert to a specific version if something fails that we cannot fix easily.
What is your take @jshipton, you have more experience on being dependent on Firedrake ;)

Reply:
I agree with @brownbaerchen - that seems to be the sanest option